test/main.js

```diff
@@ -31,6 +32,48 @@ var options = {
   templateDonut: {
     'twig': './templates/donut.twig'
   },
+  convertCategoryTitles: true,
+  convertCategoryTitlesData: {
+    "categories": {
+      "base": "00-atoms",
```

Comment:
can we apply this "00", "01" increments automatically? what if we have new categories added by someones new and he doesn't know about this place to alter converts?